src/frontend/src/test-e2e/util.ts

```diff
@@ -155,11 +155,24 @@ export async function removeVirtualAuthenticator(
   return await browser.removeVirtualWebAuth(authenticatorId);
 }
 
-export async function screenshot(
-  name: string,
-  browser: WebdriverIO.Browser
-): Promise<void> {
-  await browser.saveScreenshot(`screenshots/${name}.png`);
+// 'Screenshots' objects are used to make sure all screenshots end up in the
+// same directory, each with a different (increasing) number prefixed in the
+// filename.
+export class Screenshots {
+  private directory: string;
+  private count: number;
+
+  constructor(dir: string) {
```

Comment:
you could use "constructor(private directory:string)", then you don't need to specify the member on line 162 and assign it on 166